### PR TITLE
deleting : files; Making serverinfo work again (regional IP, no gameversion, in a table)

### PR DESCRIPTION
**Pull request by donkz**

### CODE REVIEW

File: src/pages/servers/ServerList/ServerList.module.css [hunk 1]
```diff
@@ -30,14 +28,13 @@
 
 .infoContainer {
   background-color: var(--background-light);
-  display: flex;
   flex-direction: column;
```
Comment (utrolig) [inline src/pages/servers/ServerList/ServerList.module.css:31]: If you remove `display: flex`, `flex-direction: column` won't have any effect so you might aswell remove that aswell

File: src/pages/servers/ServerList/ServerList.module.css [hunk 2]
```diff
@@ -21,7 +20,6 @@
 }
 
 .region {
-  position: absolute;
   top: 12px;
   right: 12px;
```
Comment (utrolig) [inline src/pages/servers/ServerList/ServerList.module.css:24]: If you remove `position: absolute`, `top` and `right` do nothing so just remove those aswell

File: src/pages/servers/ServerList/ServerList.tsx [hunk 3]
```diff
@@ -36,34 +38,35 @@ export const ServerList: React.FC = () => {
               );
             })
             .map((server, idx) => (
-              <div className={styles.serverInfo} key={idx}>
+			<tr>
```
Comment (utrolig) [inline src/pages/servers/ServerList/ServerList.tsx:41]: The `<tr>` elements should be wrapped by a `<tbody>` element to be valid HTML

Comment (utrolig) [inline src/pages/servers/ServerList/ServerList.tsx:41]: The `key` needs to be on the first root node returned by the `.map` function

**Review by utrolig — CHANGES_REQUESTED** [review]

**Comment by donkz** [comment]
Added a "proposed fixes" commit

File: src/pages/servers/ServerList/ServerList.tsx [hunk 4]
```diff
@@ -1,33 +1,74 @@
...
-        <div className={styles.wrapper}>
+		   <div className={styles.wrapper}>
+			   {/*
+			<div className={styles.headerRow}>
+				<div className={styles.headerCell}>Region</div>
+				<div className={styles.headerCell}>Server Name</div>
+				<div className={styles.headerCell}>IP</div>
+				<div className={styles.headerCell}><span className={styles.timestamp}>Last active</span></div>
+			</div>
+			   */}
```
Comment (vincentjr) [inline src/pages/servers/ServerList/ServerList.tsx:71]: You may want to consider handling the case where there is no data or there's an error

Comment (vincentjr) [reply src/pages/servers/ServerList/ServerList.tsx:71]: nvm i see what you're doing now

**Pull request merged**